### ISSUE TITLE
[BUG]ollama运行2.6版本，聊天报错：llama_get_logits_ith: invalid logits id 10, reason: no logits

### ISSUE THREAD

**Issue by zhb-code** [OpenBMB/MiniCPM-o#463]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

mac本地ollama部署2.6版本，按照文档部署启动ollama服务，聊天时报错：llama_get_logits_ith: invalid logits id 10, reason: no logits，请问这个问题有解决吗？

### 运行环境 | Environment

```Markdown
- OS:macos
- Python:3.11
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by luixiao0** [comment]
check #393 , openBMB's work is merging to master, you may need to compile ollama from source

**Comment by zhb-code** [comment]
直接用主分支？我试试

**Comment by tc-mb** [comment]
> @tc-mb 解决了请通知一下，谢谢！

好的，我会及时通知。

**Comment by zhb-code** [comment]
@tc-mb 解决了吗？好长时间了

**Comment by ambitiousCC** [comment]
遇到同样问题，mark一下

**Comment by tc-mb** [comment]
现在ollama应该支持了
